docs/sources/developers/plugins/add-support-for-annotations.md

```diff
@@ -81,7 +81,7 @@ const regionEvent: AnnotationEvent = {
 
 ## Build a annotation query editor
 
-Let users write custom annotation queries to only display the annotation events they care about, by adding a _query editor_.
+Let users write custom annotation queries to only display the annotation events they care about, by adding a _query editor_. You only need to build a query editor if you want to let users query or filter annotations.
 
 > **Note**: Annotation query editors have yet to receive support for React. The instructions here are given for Angular. Fortunately, you can run Angular even in a plugin otherwise written using React. This section will be updated once React support for annotation queries editors is available.
```

Comment:
```suggestion
> **Note:** Annotation query editors have yet to receive support for React. The instructions here are given for Angular. Fortunately, you can run Angular even in a plugin otherwise written using React. This section will be updated once React support for annotation queries editors is available.
```